doc/run_experiment.rst

```diff
@@ -949,9 +949,9 @@ SVR
 
 .. note::
     Note that learners not listed here do not have any default
-    parameter grids in SKLL either because there are no hyper-parameters
-    to be tuned or because there are too many and decisions about
-    which ones to tune and how depend on the data being used for the
+    parameter grids in SKLL either because either there are no
+    hyper-parameters to tune or decisions about which parameters
+    to tune (and how) depend on the data being used for the
```

Comment:
Would it be clearer to actually provide a brief direction where the user can do this tuning? If this  "best left up to the user" step is already documented in the user manual, I think it's fine. If not, perhaps cross reference the class/function where this manual tuning is can be done?

Reply:
By "best left up to the user", I just meant that the user should provide his/her own `param_grids` in the config file.

Reply:
Thanks for the clarification.